### PR TITLE
Newformat

### PR DESCRIPTION
**Pull request by cifera**
See Iss35:

'allo 'allo!

I've used the above mentioned to have "All the videos"  get the videos of the last 7 days in playable the usual list!
Its a quick and dirty fix, a bit slow to fetch the list on my pi, but better then not working at all.
Live and other functionality is still broken. Which are in use Download/Categories?

Call it version 0.5.8 maybe with better support of this api we could call it 0.6.
Help me to make it better and cleaner!

Bye :)

### CODE REVIEW

File: addon.py [hunk 1]
```diff
@@ -38,7 +38,8 @@
 # protocol : HBBTV | RTMP
 # quality  : SQ (High) | EQ (Med) | HQ (Low)
 video_json = base_url + '/papi/tvguide/videos/stream/player/{lang}/{id}_PLUS7-{lang}/{protocol}/ALL.json'
-
+video_json = 'https://api.arte.tv/api/player/v1/config/{lang}/{id}'
+daily_json = 'https://www.arte.tv/hbbtvv2/services/web/index.php/OPA/v3/programs/{date}/de'
```
Comment (koying) [inline addon.py]: Isn't the "de" for german?
Please bear with us, frenchies :)

**Comment by koying** [comment]
Thanks :)

File: addon.py (hunk 1 above)
Comment (cifera) [reply addon.py]: True! Now listing will be displayed in fr if language is set to FR.
Stream was already in fr see versionShortLibelle | "FR".

**Comment by known-as-bmf** [comment]
I refactored most of the code & deleted all code that wasnt working anymore

Coulkd tyou try this version and tell me what you think ?

Thank you very much for your work 👍

**Comment by frederic** [comment]
thank you so much guys !!

**Pull request merged**